### PR TITLE
Fix protobuf version for tests, prepare docs for 3.1.0 release

### PR DESCRIPTION
**Pull request by jonasscheid**
- Some changes/updates to the documentation before the release.
- Fix `protobuf` version to fix tests. This is necessary due to changes in the library.

### CODE REVIEW

File: epytope/doc/conf.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ def __getattr__(cls, name):
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'fred2', 'epytope Documentation',
+    (master_doc, 'eptyope', 'epytope Documentation',
```
Comment (ggabernet) [inline epytope/doc/conf.py:272]:
```suggestion
    (master_doc, 'epytope', 'epytope Documentation',
```

**Review by ggabernet — APPROVED** [review]
LGTM

**Comment by jonasscheid** [comment]
I'll add antother netmhciipan interface before the release. See #65

**Comment by christopher-mohr** [comment]
> I'll add antother netmhciipan interface before the release. See #65

👍  but I would be better to not add it to this PR

**Comment by jonasscheid** [comment]
Yes indeed, it is better to open a separate PR for this :)

**Pull request merged**